src/member/urls.py

```diff
@@ -83,24 +94,27 @@
     url(_(r'^component_to_avoid/(?P<pk>\d+)/delete/$'),
         DeleteComponentToAvoid.as_view(), name='component_to_avoid_delete'),
 
-    url(r'^client/(?P<pk>\d+)/meals/preferences$',
+    url(_(r'^client/(?P<pk>\d+)/meals/preferences$'),
         clientMealsPrefsAsJSON, name='client_meals_pref'),
 
 ]
 
 
 member_update_forms = (
-    ('basic_information', ClientUpdateBasicInformation),
-    ('address_information', ClientUpdateAddressInformation),
-    ('referent_information', ClientUpdateReferentInformation),
-    ('payment_information', ClientUpdatePaymentInformation),
-    ('dietary_restriction', ClientUpdateDietaryRestriction),
-    ('emergency_contact', ClientUpdateEmergencyContactInformation),
+    (_('basic_information'), ClientUpdateBasicInformation),
+    (_('address_information'), ClientUpdateAddressInformation),
+    (_('referent_information'), ClientUpdateReferentInformation),
+    (_('payment_information'), ClientUpdatePaymentInformation),
+    (_('dietary_restriction'), ClientUpdateDietaryRestriction),
+    (_('emergency_contact'), ClientUpdateEmergencyContactInformation),
 )
 
 # Handle client update forms URL
 for k, v in member_update_forms:
+    with translation.override('en'):
+        internal_name = str(k)
     urlpatterns.append(
-        url(_(r'^(?P<pk>\d+)/update/{}/$'.format(k)), v.as_view(),
-            name='member_update_' + k)
+        url(string_concat(_(r'^(?P<pk>\d+)/update/'), k, '/$'),
+            v.as_view(),
+            name='member_update_' + internal_name)
```

Comment:
The "internal_name" part of the route get translated, but no the 'member_update_' so we can have named URL like 'member_update_information_basique'.
You could change the implementation to translate the URL but keep route names in English?